contracts/NonFungibleToken.cdc

```diff
@@ -194,7 +197,16 @@ access(all) contract interface NonFungibleToken: ViewResolver {
 
         /// Gets the amount of NFTs stored in the collection
         /// @return An integer indicating the size of the collection
-        access(all) view fun getLength(): Int
+        access(all) view fun getLength(): Int {
+            return self.ownedNFTs.length
+        }
+
+        /// Returns an iterator that allows callers to iterate
+        /// through the list of owned NFT IDs in a collection
+        /// without having to load the entire list first
```

Comment:
Same suggestion as committed to [the FLIP](https://github.com/onflow/flips/pull/258)

```suggestion
        /// Allows a given function to iterate through the list
        /// of owned NFT IDs in a collection without first
        /// having to load the entire list into memory
```